safetensors/src/tensor.rs

```diff
@@ -849,7 +863,7 @@ mod tests {
     fn test_serialization() {
         let data: Vec<u8> = vec![0.0f32, 1.0, 2.0, 3.0, 4.0, 5.0]
             .into_iter()
-            .flat_map(|f| f.to_le_bytes())
+            .flat_map(f32::to_le_bytes)
```

Comment:
I prefer the earlier version, this one is quite important, as you can keep the loop ISO event with different dtypes.

Reply:
If you need to make this type-agnostic, the type can be left out and inference will fill it in, so `<_>::to_le_bytes` should work just as well.